src/type/directives.js

```diff
@@ -139,3 +139,40 @@ export const GraphQLSkipDirective = new GraphQLDirective({
     }
   },
 });
+
+/**
+ * Constant string used for default reason for a deprecation.
+ */
+export const DEFAULT_DEPRECATION_REASON = 'No longer supported';
+
+/**
+ * Used to declare element of a GraphQL schema as deprecated.
+ */
+export const GraphQLDeprecatedDirective = new GraphQLDirective({
+  name: 'deprecated',
+  description:
+    'Marks an element of a GraphQL schema as no longer supported.',
+  locations: [
```

Comment:
Possibly so. Right now these are the only two kinds of things in the schema which can be deprecated according to the spec.